### PR TITLE
GIVCAMP-278 | Image Test

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# NOT READY FOR REVIEW
- (Edit the above to reflect status)

# Summary
- TL;DR - what's this PR for?

# Review By (Date)
- When does this need to be reviewed by?

# Criticality
- How critical is this PR on a 1-10 scale? Also see [Severity Assessment](https://stanfordits.atlassian.net/browse/D8CORE-1705).
- E.g., it affects one site, or every site and product?

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch or go to the netlify preview
2. Navigate to...
3. Verify...

# Associated Issues and/or People
- JIRA ticket(s) - GIVCAMP-
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

### CODE REVIEW

File: app/(storyblok)/loading.tsx [hunk 1]
```diff
@@ -9,18 +9,15 @@ const Loading = () => {
...
-        <Container width='site' className='rs-my-8'>
+        <Container width='site' className='rs-my-10'>
           <FlexBox gap direction='row' wrap='wrap'>
-            <Skeleton height={300} highlightColor='rgba(67,66,62,.8)' containerClassName='w-full'/>
-            <Skeleton height={300} highlightColor='rgba(67,66,62,.8)' containerClassName='min-w-[25%] flex-grow' />
-            <Skeleton height={300} highlightColor='rgba(67,66,62,.8)' containerClassName='min-w-[25%] flex-grow' />
-            <Skeleton height={300} highlightColor='rgba(67,66,62,.8)' containerClassName='min-w-[25%] flex-grow' />
-            <Skeleton height={200} highlightColor='rgba(67,66,62,.8)' containerClassName='min-w-[40%] flex-grow' />
+            <Skeleton height={500} highlightColor="rgba(255,255,255,0.2)" baseColor="rgba(0,0,0,0.6)" containerClassName='w-full'/>
+            <Skeleton height={300} highlightColor="rgba(255,255,255,0.2)" baseColor="rgba(0,0,0,0.6)" containerClassName='w-full'/>
```
Comment (yvonnetangsu) [inline app/(storyblok)/loading.tsx:15]: @sherakama 😂

Comment (sherakama) [reply app/(storyblok)/loading.tsx:15]: Uh, yeah, I'm totally messing with those in https://github.com/SU-SWS/ood-giving-campaign/pull/200.

I was having issues with https://www.npmjs.com/package/react-loading-skeleton and the way they exported the package so I switched over to https://github.com/danilowoz/react-content-loader

I think the options in there look nice.

Screenshot 2024-01-24 at 09 14 37

Comment (sherakama) [reply app/(storyblok)/loading.tsx:15]: Keep going on your PR and I'll adjust later. The PR I have is for the future stuff so we don't have to change right away.

Comment (yvonnetangsu) [reply app/(storyblok)/loading.tsx:15]: Oh Cool - thanks for the heads up 😄

**Comment by yvonnetangsu** [comment]
I decided to not go with Next Image at the moment, but trying other ways for further optimization (plus fixed another small thing that I've been meaning to get to with the story Hero Image on SM breakpoint).

**Comment by yvonnetangsu** [comment]
Closing this

**Pull request closed without merging**